data/datasets.md

```diff
@@ -141,3 +141,110 @@ To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the follo
+
+* `mrlEyes_2018_01` - directory containing subdirectories with dataset images
+
+### Datasets in dataset_definitions.yml
+* `mrlEyes_2018_01` used for evaluation models on MRL Eye dataset for recognition of eye state. (model examples: [`open-closed-eye-0001`](../models/public/open-closed-eye-0001/README.md))
+
+## [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/)
+
+### How download dataset
+
+To download LFW dataset, you need to follow the steps below:
+1. Go to the [Labeled Faces in the Wild](http://vis-www.cs.umass.edu/lfw/) website
+2. Go to the `Download the database` section
+3. Select `All images as gzipped tar file` and download archive
+4. Unpack archive
+5. Go to the `Training, Validation, and Testing` section
+6. Select `pairs.txt` and download pairs file
+7. Download [`lfw_landmark`](https://raw.githubusercontent.com/clcarwin/sphereface_pytorch/master/data/lfw_landmark.txt) file
+
+### Files layout
+
+To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the following:
+
+* `LFW` - directory containing images directories, pairs and landmarks files
+    * `lfw` - directory containing the LFW images
+    * `annotation` - directory containing pairs and landmarks files
+        * `pairs.txt` - file with annotation positive and negative pairs for LFW dataset
+        * `lfw_landmark.txt` - file with facial landmarks coordinates for annotation images of LFW dataset
+
+### Datasets in dataset_definitions.yml
+* `lfw` used for evaluation models on LFW dataset for face recognition task. (model examples: [`Sphereface`](../models/public/Sphereface/README.md))
+
+## [NYU Depth Dataset V2](https://cs.nyu.edu/~silberman/datasets/nyu_depth_v2.html)
+
+### How download dataset
+
+To download NYU Depth Dataset V2 preprocessed data stored in HDF5 format, you need to follow the steps below:
+1. [Download](http://datasets.lids.mit.edu/fastdepth/data/nyudepthv2.tar.gz) archive from the [website](http://datasets.lids.mit.edu/fastdepth/data/)
+2. Unpack archive
+
+### Files layout
+
+To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the following:
+
+* `nyudepthv2/val` - directory with dataset official data and converted images and depth map
+    * `official` - directory with data stored in original hdf5 format
+    * `converted` - directory with converted data
+        * `images` - directory with converted images
+        * `depth` -  directory with depth maps
+
+Note: If dataset is used in the first time, please set `allow_convert_data: True` in annotation conversion parameters for this dataset in `dataset_definitions.yml`  or use [convert.py](../tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py) and  following command line to get converted data .
+
+```sh
+OMZ_ROOT/accuracy_checker/accuracy_checker/annotation_converters/convert.py  nyu_depth_v2 --data_dir <DATASET_DIR>/nyudepthv2/val/official --allow_convert_data True
```

Comment:
there is entry point for conversion (like `accuracy_check`)
```suggestion
convert_annotation  nyu_depth_v2 --data_dir <DATASET_DIR>/nyudepthv2/val/official --allow_convert_data True
```

Reply:
Done